app/controllers/transactions_controller.rb

```diff
@@ -22,6 +22,7 @@ def new
       else
         @transaction = @transaction.dup
         @transaction.amount = @transaction.amount.abs
+        @transaction.date = DateTime.now
```

Comment:
@psmir it's better to use `Time.current`, cos it uses current timezone, instead of server timezone.